sunpy/time/timerange.py

```diff
@@ -460,3 +461,20 @@ def intersects(self, other):
             int_second = self
 
         return int_second.start <= int_first.end
+
+    def range_list(self, frequency):
+        """
+        Return a Linearly Spaced List of Astropy Time Objects between the passed TimeRange.
+
+        Parameters
+        ----------
+        frequency : `int`
+            The number of objects needed
+        """
+        if frequency<=0:
+            raise ValueError("The number of objects needed cannot be 0 or less")
+        end = self._t2
+        start = self._t1
+        delta = (end-start)
+        time_list = list(start + delta * np.linspace(0,1,frequency))
```

Comment:
It should be possible to return one astropy.time.Time object that has a range of values, I would personally prefer that instead of a list.